src/lib/process-results/index.js

```diff
@@ -1,5 +1,3 @@
-import chalk from 'chalk';
```

Comment:
I removed chalk from this file, as the generated strings are sent to the UI to be displayed – where we don't show color.